docs/platform/howto/_manage-org-vpc-peering-azure.md

```diff
@@ -0,0 +1,426 @@
+   permissions to peer your VNet.
+
+   :::note
+   Save the `id` field from the JSON output. It will be referred to as `$user_sp_id`.
+   :::
+
+1. Set a password for your app object:
+
+   ```bash
+   az ad app credential reset --id $user_app_id
+   ```
+
+   :::note
+   Save the `password` field from the output. It will be referred to as `$user_app_secret`.
+   :::
+
+1. Find properties of your virtual network:
+
+   - Resource ID
+     - In the [Azure portal](https://portal.azure.com/#home): **Virtual networks** >
+       name of your network > **JSON View** > **id**
+     - Using the Azure CLI:
+
+       ```bash
+       az network vnet list
+       ```
+
+     :::tip
+     The `id` field should have format
+     `/subscriptions/$user_subscription_id/ resourceGroups/$user_resource_group/providers/Microsoft.Network/virtualNetworks/$user_vnet_name`. It will be referred to as `$user_vnet_id`.
+
+   - Azure Subscription ID (the VNet page in the
+     [Azure portal](https://portal.azure.com/#home) > **Essentials** > **Subscription ID**)
+     or the part after `/subscriptions/` in the resource ID. It will be referred to as
+     `$user_subscription_id`.
+   - Resource group name (the VNet page in the [Azure portal](https://portal.azure.com/#home) >
+     **Essentials** > **Resource group**) or the `resourceGroup` field in the output. This
+     will be referred to as `$user_resource_group`.
+   - VNet name (title of the VNet page), or the `name` field from
+     the output. It will be referred to as `$user_vnet_name`.
+
+   :::note
+   Save all the properties for later.
+   :::
+
+1. Grant your service principal permissions to peer.
+
+   The service principal needs to be assigned a role that has the permission for the
+   `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write` action
+   on the scope of your VNet. To limit the amount of permissions the application
+   object and the service principal have, you can create a custom role with just
+   that permission. The built-in network contributor role includes that permission.
+
+   1. Find the id of the role with the required permission:
+
+      ```bash
+      az role definition list --name "Network Contributor"
+      ```
+
+      The `id` field from the output will be referred to as `$network_contributor_role_id`.
+
+   1. Assign the service principal the network contributor role using
+      `$network_contributor_role_id`:
+
+      ```bash
+      az role assignment create             \
+        --role $network_contributor_role_id \
+        --assignee-object-id $user_sp_id    \
+        --scope $user_vnet_id
+      ```
+
+      This allows your application object to manage the network in the `--scope`.
+      Since you control the application object, it may also be given permission for the
+      scope of an entire resource group or the whole subscription to allow creating other
+      peerings later without assigning the role for each VNet separately.
```

Comment:
For added clarity: 

This allows your application object to manage the network within the specified `--scope`. Since you control the application object, you can also grant it permissions at the scope of an entire resource group or the whole subscription. This enables creating other peerings later without assigning the role to each VNet separately.